yt/python/yt/cli/strawberry_parser.py

```diff
@@ -8,6 +8,17 @@
 import os
 
 
+def _normalize_proxy(proxy):
+    for prefix in ["http://", "https://"]:
+        if proxy.startswith(prefix):
+            proxy = proxy[len(prefix):]
```

Comment:
I can't you [removeprefix](https://docs.python.org/3/library/stdtypes.html#str.removeprefix) here due to python 3.7 :(